EleksTubeHAX_pio/src/GLOBAL_DEFINES.h

```diff
@@ -372,12 +371,45 @@
   // #define CSSR_LATCH_PIN (-1)
 
   // All IPSTUBEs has the LCDs pins VCC power (LED Anode) and VDD (Power Supply for Analog) connectected to the VCC (3.3V) and Ground to Ground (PCB), so the displays are Always-On!
-  // EXCEPT: The Q1 transistor is present! 
+  // EXCEPT: The Q1 transistor is present!
   // Then the GPIO4 pin is connected to the transistor and Ground of the LCDs is running through the transistor, so the LCDs can be turned on and off AND dimmed!
   #define TFT_ENABLE_PIN (GPIO_NUM_4) // pin 24 is GPIO4
   //if transistor is present and we want hardware dimming, we need to choose a PWM channel for this, can always be defines, even if not used
   #define TFT_PWM_CHANNEL 0
 
+  // Skip reinitialization
+  // This feature is only for IPSTUBE clocks by now!!! Can also be used on other clocks, but not tested yet!
+  // Always skip reinitialization for IPSTUBE clocks, because the displays are either always on (versions without Q1 transistor) 
+  // and a reinit just shows strange patterns on the displays and forces an unnecessary redraw of the clock digits.
+  // or seems to don't need a reinit (with Q1) -> this is an assumption, because the display came back on after a few hours without problems
+  // NOTE: If this causes wake up issues for you, disable it by commenting the following line out and go back to full reinit after display power off
+  #define TFT_SKIP_REINIT
+
+  // Hardware dimming!
+  // This feature is only supported by IPSTUBE clocks by now!!!
+  // DON'T USE IT WITH OTHER CLOCKS! IT MAY DAMAGE YOUR CLOCK!
+
+  // In case you have an IPSTUBE clock that does not support hardware dimming because of missing Q1 transistor:
+  // This will NOT damage your clock, but the dimming of the displays will be totally disabled! Also the LCD power switch will not work!
+  // If you notice, that the night time dimming or manual dimming does not work, you will have a clock without the Q1 transistor 
+  // and you can/should comment the following line out to get back to the software dimming!
+
+  // Comment the next line out, to DISABLE hardware dimming with GPIO4 pin (TFT_ENABLE_PIN) for a IPSTUBE clock  
+  #define DIM_WITH_ENABLE_PIN_PWM
```

Comment:
It might be beneficial to more follow the "user - global" defines pattern to either add a define that can be used to _disable_ this define or add a commented out `#undef DIM_WITH_ENABLE_PIN_PWM` to empty user defines.

Reply:
totally agreed! Will add this